tests/Api/ProductMediaFileApiIntegration.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace Akeneo\Pim\tests\Api;
+
+use Akeneo\Pim\Api\ProductMediaFileApi;
+use Akeneo\Pim\tests\MediaSanitizer;
+
+class ProductMediaFileApiIntegration extends ApiTestCase
+{
+    public function testListPerPage()
```

Comment:
By putting it in a dedicated file, you can do it as in the PIM. It will be better to test list: next page, previous page.

https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/ApiBundle/tests/integration/Controller/Attribute/ListAttributeIntegration.php#L35